server/recceiver/cfstore.py

```diff
@@ -20,37 +19,244 @@
 from twisted.internet.threads import deferToThread
 
 from . import interfaces
+from .interfaces import CommitTransaction
+from .processors import ConfigAdapter
 
 _log = logging.getLogger(__name__)
 
-# ITRANSACTION FORMAT:
-#
-# source_address = source address
-# records_to_add = records ein added ( recname, rectype, {key:val})
-# records_to_delete = a set() of records which are being removed
-# client_infos = dictionary of client client_infos
-# record_infos_to_add = additional client_infos being added to existing records
-# "recid: {key:value}"
-#
-
 __all__ = ["CFProcessor"]
 
-RECCEIVERID_KEY = "recceiverID"
 RECCEIVERID_DEFAULT = socket.gethostname()
+DEFAULT_MAX_CHANNEL_NAME_QUERY_LENGTH = 600
+
+
+class PVStatus(enum.Enum):
+    """PV Status values."""
+
+    Active = enum.auto()
+    Inactive = enum.auto()
```

Comment:
Enum states are typically ALL_CAPS

Reply:
Thinking more about this, `StrEnum` is def the way to go - don't think we have any other option for `CFPropertyName`.